styles.css

```diff
@@ -512,6 +506,10 @@ span.unread {
     flex-grow: 1;
 }
 
+.moodleoverflowpost .questioner {
+    background-color: rgba(0, 127, 255, 0.1);
```

Comment:
The highlighting is a good idea but can you make it a bit more subtle? I feel like the color is a bit too strong so that one automatically only focuses on the name and not the content.

Reply:
I made the color a bit lighter :)